File: API/Features/Billing/Codes/ViewModels/CodeListVM.cs
using System;

namespace API.Features.Billing.Codes {

    public class CodeListVM {

        public Guid Id { get; set; }
        public string Description { get; set; }
        public string Batch { get; set; }
        public string LastDate { get; set; }
        public int LastNo { get; set; }
        public bool IsActive { get; set; }
        public string Customers { get; set; }
        public string Suppliers { get; set; }
        public bool IsMyData { get; set; }
        public string Table8_1 { get; set; }
        public string Table8_8 { get; set; }
        public string Table8_9 { get; set; }

    }

}

File: FrontEnd/src/app/features/billing/codes/classes/view-models/code-list-vm.ts
import { Guid } from 'guid-typescript'

export interface CodeListVM {

    id: Guid
    description: string
    batch: string
    lastDate: string
    formattedLastDate: string
    lastNo: number
    isActive: boolean
    // Plus or Minus
    customers: string
    suppliers: string
    // myData
    isMyData: boolean
    table8_1: string
    table8_8: string
    table8_9: string

}
